sbb_newspapers_org_image/__init__.py

```diff
@@ -1 +1 @@
-from eynollah import *
+
```

Comment:
Does this belong in this PR?

Reply:
Either this PR or the refactoring PR. Top-level wildcard import is suspicious here and probably unintentional.

Reply:
OK